addons/portal/controllers/portal.py

```diff
@@ -308,10 +308,65 @@ def account(self, redirect=None, **post):
     def on_account_update(self, values, partner):
         pass
 
+    def _get_commercial_partners_and_users(self):
```

Comment:
I think it can be inlined directly as there are two calls (and two times same route -> ?) Having a method that returns two stuff is a bit strange... in the end, it is easier to understand code by inlining it directly.